src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudFoundryApplication.java

```diff
@@ -43,9 +43,10 @@
 @ImplementedBy(VanillaCloudFoundryApplicationImpl.class)
 public interface VanillaCloudFoundryApplication extends Entity, Startable, DriverDependentEntity {
 
-    @SetFromFlag("name")
-    ConfigKey<String> APPLICATION_NAME = ConfigKeys.newStringConfigKey(
-            "cloudFoundry.application.name", "Name of the application");
+    @SetFromFlag("name-app")
```

Comment:
It does not look possible. This flag `name` is added to the entity as the `displayName` in case of the `displayName`.

There is not any `ConfigKey` for getting the `name`, at least I did not found it using the debugger. [Here](https://github.com/apache/brooklyn-server/blob/master/camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/creation/BrooklynComponentTemplateResolver.java#L232) you can see how camp resolver adds the `name` as the `displayName` of the entity.